web_app/config.py

```diff
@@ -8,3 +9,10 @@
 except KeyError:
     print('не найдена переменная')
     raise
+
+DATABASE = 'postgres'
```

Comment:
Пусть это в env живёт, и хост с портом тоже. Можно сделать дефолтные значения через getenv
```suggestion
DATABASE = os.getenv('DB_NAME', 'postgres')
```